packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -136,12 +143,13 @@ public void onAuthenticationError(int errorCode, CharSequence errString) {
 
   @Override
   public void onAuthenticationSucceeded(BiometricPrompt.AuthenticationResult result) {
+    Log.d("LocalAuth", "Authentication Succeed");
     stop(true);
   }
 
   @Override
   public void onAuthenticationFailed() {
-    stop(false);
+    Log.d("LocalAuth", "Authentication Failed");
```

Comment:
Could you provide the reasoning here? (What you put in the PR description as code comments).

Reply:
Sorry, couldn't get you. Reason for what ?